sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -537,18 +552,20 @@ static inline bool _in_abrs(const _nib_abr_entry_t *abr)
 
 void _nib_offl_clear(_nib_offl_entry_t *dst)
 {
-    if (dst->next_hop != NULL) {
-        _nib_offl_entry_t *ptr;
-        for (ptr = _dsts; _in_dsts(ptr); ptr++) {
-            /* there is another dst pointing to next-hop => only remove dst */
-            if ((dst != ptr) && (dst->next_hop == ptr->next_hop)) {
-                break;
+    if (dst->mode == _EMPTY) {
```

Comment:
This changes the semantics from "Clear off link entry" to "Clear off link entry only if removed from every view"... So please give your reasoning for this. Maybe the the function was used wrong somewhere?

Reply:
Assume it was not `_EMPTY`. Simply `memsetting` it could be fatal.

Reply:
I'd argue: then don't call this function, but ok, let's go this route. But no guarantee if this introduces new bugs...